code/game/dna/dna2_helpers.dm

```diff
@@ -151,11 +150,13 @@
 
 		H.s_tone   = 35 - dna.GetUIValueRange(DNA_UI_SKIN_TONE, 220) // Value can be negative.
 
-		if(S.has_gender)
-			if(dna.GetUIState(DNA_UI_GENDER))
-				H.change_gender(FEMALE, 0)
-			else
+		switch(dna.GetUITriState(DNA_UI_GENDER))
+			if(0)
+				H.change_gender(FEMALE,0)
```

Comment:
spacing and `FALSE` instead of `0`. Boolean values should be set to either `TRUE` or `FALSE`. This improves readability.
Same for the ones below

Reply:
Mmm. Gotcha!